lib/db/DB.ts

```diff
@@ -53,6 +53,11 @@ class DB {
 
   async init() {
     try {
+      const initSequelize = new Sequelize('', this.sequelize.options.username, this.sequelize.options.password, this.sequelize.options);
```

Comment:
Move this to a catch block to only fire when we get an error on accessing a non-existing database, see my last review comment above on how I'd approach this.